setup.py

```diff
@@ -5,10 +5,20 @@
 
 INSTALL_REQUIREMENTS = [
     "Django>=1.11,<2.0",
-    "django-cms>=3.5.0",
+    'django_cms',
     "django-treebeard>=4.3",
 ]
 
+TESTS_REQUIRE = [
+    'djangocms_versioning',
+    'djangocms_version_locking',
+
```

Comment:
Seems to be a blank space here.

Reply:
Yeah, the reason for the blank space is that I actually had a question around that. To me it looks like this package really requires both of these packages for it to work properly. So the question is whether this should be moved out from `TESTS_REQUIRE` to `INSTALL_REQUIREMENTS`.